### ISSUE TITLE
Stream Deck Mini "caches" Keys after running animated example

### ISSUE THREAD

**Issue by michaelkleinhenz** [abcminiuser/python-elgato-streamdeck#17]
This issue might be related to #13, but shows a different behaviour.

The first 2 keypresses are registered to the previous key. For example, if I run `example_basic.py` and press key #3 for 4 times with a registered press on #1 before:
```
Deck b'0001:0059:00' Key 1 = True
Deck b'0001:0059:00' Key 1 = False
Deck b'0001:0059:00' Key 1 = True
Deck b'0001:0059:00' Key 1 = False
Deck b'0001:0059:00' Key 3 = True
Deck b'0001:0059:00' Key 3 = False
```
Note that the first 2 presses on #3 are registered as #1. This can be repeated and it is always the first 2 presses that are being registered incorrectly.

On the *first* press after launching `example_basic.py`, it looks like a random "previous" key is chosen.

In addition to the above issue, the deck sometimes registers random key presses inbetween correct ones or does not register presses at all. From my experiments, this applies to keys #4 and #5 (the last two keys) more often than to the first 4 keys.

Here's the deck info:
```
Deck 0 - Stream Deck Mini.
	 - ID: b'0001:0059:00'
	 - Serial: BL44H1B15985
	 - Firmware Version: 3.00.000
	 - Key Count: 6 (2x3 grid)
	 - Key Image Format: 80x80 pixels, BGR order, rotated 90 degrees, not mirrored
```

**Comment by michaelkleinhenz** [comment]
Ok, this is weird. Did more experiments and it looks like it's not the USB hub but the animated example. The deck works before running the animation example, but not after running it. Also, the animation "flickers", similar to what is described in #13

**Comment by michaelkleinhenz** [comment]
This was also fixed with the fix for #13.
